website/thunderbird/102.0/whatsnew/index.html

```diff
@@ -13,15 +13,14 @@ <h1 class="font-4xl md:font-hero font-thin md:mt-10 mb-8">{{ self.page_title() }
       <p class="font-lg md:font-xl tracking-wide leading-relaxed max-w-4xl lg:mb-16 mb-8 mt-0 p-links">
         {{ _('This article describes some of the major changes visible to users in Thunderbird version 102. Full details of all the changes can be found in the <a href="%(release)s">Thunderbird release notes from 102.0 and up</a>.')|format(release=url('thunderbird.releases.index')) }}
       </p>
-      <p class="max-w-lg tracking-wider leading-loose mb-6 mt-0">
-        {{ _('Thunderbird is both free and freedom respecting, but we’re also completely funded by donations! Help us sustain the project and continue to improve.') }}
-      </p>
 
-      <div class="flex mb-20">
-        <a href="{{ donate_url('home_page_body') }}" class="btn-donate btn-donate-whatsnew font-lg">
-          <span class="inline-block w-4 mr-1">{{ svg('heart') }}</span> {{_('Donate')}}
-        </a>
-      </div>
+      <a href="{{ donate_url('home_page_body') }}" class="donation-appeal">
+        <div class="inline-block w-16 mr-4 text-donation">{{ svg('heart') }}</div>
+        <p class="tracking-wider leading-loose m-0">
```

Comment:
I'd use a slightly tighter kerning and line height.
```suggestion
        <p class="tracking-wide leading-normal m-0">
```